boot/makebootable.go

```diff
@@ -327,21 +346,14 @@ func makeRunnableSystem(model *asserts.Model, bootWith *BootableSet, sealer *Tru
 	if err := os.MkdirAll(snapBlobDir, 0755); err != nil {
 		return err
 	}
-	for _, fn := range []string{bootWith.BasePath, bootWith.KernelPath, bootWith.GadgetPath} {
-		dst := filepath.Join(snapBlobDir, filepath.Base(fn))
```

Comment:
We probably don't need this change, `snap prepare-image` should put the right filename already into the seed